services/frontend-service/src/ui/utils/store.tsx

```diff
@@ -487,7 +487,9 @@ export const useCurrentlyDeployedAtGroup = (application: string, version: number
 // Get release information for a version
 export const useReleaseInfo = (app: string, version: number): Release =>
     useOverview(({ applications }) => {
-        const releaseInfo = applications[app]?.releases.filter((release) => release.version === version)[0];
+        const releaseInfo = applications[app]?.releases.filter((r) =>
+            version === -1 ? r.undeployVersion : r.version === version
```

Comment:
do we really need to handle "-1" here?
I'd like to get red of all "-1".

Reply:
done :+1: